Makefile

```diff
@@ -1,9 +1,21 @@
 
 PROFILES = nodeps alldeps
 
-TEST_ALL:
+ALL: default dalston finchley classpath
+
+default:
 	@./mvnw clean install
+
+# is failing, it does not know artemis dependencies
+angel:
+	@./mvnw clean test -Dversion.org.springframework.boot=1.2.8.RELEASE -Dversion.org.springframework.cloud-spring-cloud-dependencies=Angel.SR6
+
+dalston:
+	@./mvnw clean test -Dversion.org.springframework.boot=1.5.6.RELEASE -Dversion.org.springframework.cloud-spring-cloud-dependencies=Dalston.SR3
+
```

Comment:
camden is missing here but there is an angel?

Reply:
does it make sense to add  `Edgware.M1`, and `Finchley.M2` (to start testing?)

Reply:
thanks. if my app uses `dalston` should I override `version.org.springframework.cloud-spring-cloud-dependencies` property in my `pom.xml`?

Reply:
no why would you do it? 

Just use the released version which is compatible with C and D.

Reply:
ok, thanks